indy-besu/vdr/src/contracts/auth/role.rs

```diff
@@ -17,16 +17,16 @@ pub enum Role {
 pub type HasRole = bool;
 pub type RoleIndex = u8;
 
-impl Into<ContractParam> for Role {
-    fn into(self) -> ContractParam {
-        trace!("Role: {:?} convert into ContractParam has started", self);
+impl From<Role> for ContractParam {
```

Comment:
Could you tell why we change this?

Reply:
Implementing `From` you can call in two ways: `ContractParam::from` and `Role.into()` while implementing Into you can call only one way

Reply:
Got it. Thank you!